src/javascript/app/pages/trade/tradepage.js

```diff
@@ -57,6 +58,22 @@ const TradePage = (() => {
                     if ($currency.next().attr('id') === $currency.attr('id')) $currency.next().eq(0).remove();
                 }
             });
+
+            BinarySocket.wait('website_status', 'landing_company').then(() => {
+                const is_logged_in = Client.isLoggedIn();
+                const is_eu = isEuCountry();
+                const is_maltainvest = Client.get('landing_company_shortcode') === 'maltainvest';
+
+                if (is_maltainvest) {
```

Comment:
@aaron-binary @negar-binary  we also need to keep this hidden for `MF professionals`

Reply:
yes, thank you